dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/program/DefaultProgramIndicatorService.java

```diff
@@ -72,34 +72,30 @@
-        put( MonthsBetweenSqlFunction.KEY, new MonthsBetweenSqlFunction() ).
-        put( YearsBetweenSqlFunction.KEY, new YearsBetweenSqlFunction() ).
-        put( ConditionalSqlFunction.KEY, new ConditionalSqlFunction() ).
-        put( HasValueSqlFunction.KEY, new HasValueSqlFunction() ).build();
-    
-    private static final Map<String, String> VARIABLE_SAMPLE_VALUE_MAP = ImmutableMap.<String, String>builder().
-        put( ProgramIndicator.VAR_COMPLETED_DATE, "'2017-07-08'" ).
-        put( ProgramIndicator.VAR_CURRENT_DATE, "'2017-07-08'" ).
-        put( ProgramIndicator.VAR_DUE_DATE, "'2017-07-08'" ).
-        put( ProgramIndicator.VAR_ENROLLMENT_COUNT, "1" ).
-        put( ProgramIndicator.VAR_ENROLLMENT_DATE, "'2017-07-08'" ).
-        put( ProgramIndicator.VAR_ENROLLMENT_STATUS, "'COMPLETED'" ).
-        put( ProgramIndicator.VAR_EVENT_COUNT, "1" ).
-        put( ProgramIndicator.VAR_EVENT_DATE, "'2017-07-08'" ).
-        put( ProgramIndicator.VAR_EXECUTION_DATE, "'2017-07-08'" ).
-        put( ProgramIndicator.VAR_INCIDENT_DATE, "'2017-07-08'" ).
-        put( ProgramIndicator.VAR_ANALYTICS_PERIOD_START, "'2017-07-08'" ).
-        put( ProgramIndicator.VAR_PROGRAM_STAGE_ID, "'WZbXY0S00lP'" ).
-        put( ProgramIndicator.VAR_PROGRAM_STAGE_NAME, "'First antenatal care visit'" ).
-        put( ProgramIndicator.VAR_TEI_COUNT, "1" ).
-        put( ProgramIndicator.VAR_VALUE_COUNT, "1" ).
-        put( ProgramIndicator.VAR_ZERO_POS_VALUE_COUNT, "1" ).build();
+    private static final Map<String, SqlFunction> SQL_FUNC_MAP = ImmutableMap.<String, SqlFunction> builder()
+        .put( ZeroIfNegativeSqlFunction.KEY, new ZeroIfNegativeSqlFunction() )
+        .put( OneIfZeroOrPositiveSqlFunction.KEY, new OneIfZeroOrPositiveSqlFunction() )
+        .put( ZeroPositiveValueCountFunction.KEY, new ZeroPositiveValueCountFunction() )
+        .put( DaysBetweenSqlFunction.KEY, new DaysBetweenSqlFunction() )
+        .put( WeeksBetweenSqlFunction.KEY, new WeeksBetweenSqlFunction() )
+        .put( MonthsBetweenSqlFunction.KEY, new MonthsBetweenSqlFunction() )
+        .put( YearsBetweenSqlFunction.KEY, new YearsBetweenSqlFunction() )
+        .put( ConditionalSqlFunction.KEY, new ConditionalSqlFunction() )
+        .put( HasValueSqlFunction.KEY, new HasValueSqlFunction() ).build();
+
+    private static final Map<String, String> VARIABLE_SAMPLE_VALUE_MAP = ImmutableMap.<String, String> builder()
+        .put( ProgramIndicator.VAR_COMPLETED_DATE, "'2017-07-08'" )
+        .put( ProgramIndicator.VAR_CURRENT_DATE, "'2017-07-08'" ).put( ProgramIndicator.VAR_DUE_DATE, "'2017-07-08'" )
+        .put( ProgramIndicator.VAR_ENROLLMENT_COUNT, "1" ).put( ProgramIndicator.VAR_ENROLLMENT_DATE, "'2017-07-08'" )
+        .put( ProgramIndicator.VAR_ENROLLMENT_STATUS, "'COMPLETED'" ).put( ProgramIndicator.VAR_EVENT_COUNT, "1" )
+        .put( ProgramIndicator.VAR_EVENT_DATE, "'2017-07-08'" )
+        .put( ProgramIndicator.VAR_EXECUTION_DATE, "'2017-07-08'" )

```

Comment:
Is this production code or test code?

Reply:
This is sample values that is used for validating expression format.